pipeline-runner/tests/testthat/test-gem2s-6-construct_qc_config.R

```diff
@@ -13,6 +13,9 @@ mock_scdata <- function() {
   # add doublet scores
   scdata$doublet_scores <- rep(c(0.01, 0.9), each = 40)
   scdata$doublet_class <- rep(c("singlet", "doublet"), each = 40)
+
+  # add mitochondrial percent
+  scdata$percent.mt <- rnorm(ncol(scdata), mean = 6)
```

Comment:
Using rnorm to reproduce percent.mt data will likely give no outliers --> the outliers method works better with a gamma distribution.

I appreciate it doesn't really matter in this test :)
Just asking to make sure I'm understanding the outliers method properly.

Reply:
This was added because with the new method if the `percent.mt` slot is absent it fails. This is something we enforce if MT genes are not detected (it is set to 0)